website/docs/r/batch_job_definition.html.markdown

```diff
@@ -0,0 +1,78 @@
+---
+layout: "aws"
+page_title: "AWS: batch"
+sidebar_current: "docs-aws-resource-batch-job-definition"
+description: |-
+  Provides a Batch Job Definition resource.
+---
+
+# aws_batch_job_definition
+
+Provides a Batch Job Definition resource.
+
+## Example Usage
+
+```hcl
+resource "aws_batch_job_definition" "test" {
+	name = "tf-test-batch-job-definition"
```

Comment:
I think we should replace hyphens with underscores here, per validation function.